### ISSUE TITLE
Add type annotations?

### ISSUE THREAD

**Issue by d33tah** [dr-prodigy/python-holidays#560]
I imported the library into my project and the CI warned me:

```
+ dmypy run ksiemgowy -- --strict
ksiemgowy/bookkeeping.py:14: error: Skipping analyzing "holidays": found module but no type hints or library stubs
ksiemgowy/bookkeeping.py:14: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

Depending on the amount of effort, maybe it's worth to add type annotations? In some interesting cases, this might unlock new superpowers, such as being able to better JIT the code.

**Comment by dr-prodigy** [comment]
Hey there @d33tah
thank you for your suggestion!

This is an interesting code improvement, although I expect a significant effort to put it in place.

If you are interested in working on this feature implementing a PR, we will do our best to support you.. Pls let me know, thx!

Cheers
dr-p

**Comment by d33tah** [comment]
Actually it does look like quite a few hours of work. I'm afraid might not have time to do that in the near future, sorry. It might be a good idea to leave the issue hanging though.

**Comment by mborsetti** [comment]
I think this should be fixed once #608 is merged.
